testsuite/integration-arquillian/servers/auth-server/services/testsuite-providers/src/main/java/org/keycloak/testsuite/rest/TestingResourceProvider.java

```diff
@@ -690,6 +691,20 @@ public TestingExportImportResource getExportImportResource() {
                         }));
     }
 
+    @GET
+    @Path("/test-amhibian-component")
```

Comment:
Typo in the URL in "amhibian" . This may be irrelevant if we're going to rename AmphibianProviderFactory.